inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -355,6 +355,7 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
     bool recalc_processing_order = false;
     std::map<primitive_id, std::vector<primitive_id>> fusing_history;
 
+    const uint8_t supports_immad = p.get_engine().get_device_info().convert_to_api().supports_immad;
```

Comment:
Why do you call `convert_to_api()`? To my mind it's not needed

Reply:
Fixed it. thanks